haystack/schema.py

```diff
@@ -94,17 +96,52 @@ def __str__(self):
         return str(self.to_dict())
 
 
+@dataclass
+class Answer:
+    answer: Optional[str]
+    type: Literal["generative", "extractive"]
+    #question: Optional[str]
+    score: Optional[float]
+    #probability: Optional[float] = None
+    context: Optional[str]
+    offset_start: Optional[int]
+    offset_end: Optional[int]
+    offset_start_in_doc: Optional[int]
+    offset_end_in_doc: Optional[int]
```

Comment:
Just curious what will be the values for TabularQA, VisualQA etc

Reply:
We decided to go for something more generic now (see latest code). In TableQA the ints will represent the cells in the table (start cell, end cell). We thought about going two-dimensional (x and y coordinates) but felt that it makes the handling of regular QA cases very inconvenient.